Description: Test for usb-db helper before running it
 In newer udev versions, the usb-db udev helper has been replaced by
 the hwdb builtin. Instead of removing the import, test for the helper
 before running it. This way we are compatible with both old and new
 udev.
Author: Michael Biebl <biebl@debian.org>
--- argyll-1.4.0.orig/libusb1/55-Argyll.rules
+++ argyll-1.4.0/libusb1/55-Argyll.rules
@@ -59,6 +59,6 @@ TEST=="/lib/udev/udev-acl", TEST=="/var/
 ENV{COLOR_MEASUREMENT_DEVICE}=="*?", ENV{ACL_MANAGE}!="*?", MODE="660", GROUP="plugdev"
 
 # Set ID_VENDOR and ID_MODEL acording to VID and PID
-IMPORT{program}="usb-db %p"
+TEST=="/lib/udev/usb-db", IMPORT{program}="usb-db %p"
 
 LABEL="argyll_rules_end"
